test/benchmarks/material/button-harness/BUILD.bazel

```diff
@@ -0,0 +1,82 @@
+load("@npm_angular_dev_infra_private//benchmark/component_benchmark:component_benchmark.bzl", "component_benchmark")
+load("//tools:defaults.bzl", "ng_test_library", "ng_web_test_suite", "ts_library")
+
+package(default_visibility = ["//visibility:public"])
+
+# These are two separate files that provide the same tidy interface for running a performance
+# benchmark.
+#
+# I am not sure why the testbed version of the test does not log console.time to the terminal,
+# although I assume it has something to do with the default karma config that is generated by
+# karma_web_test_suite (which is wrapped by ng_web_test_suite).
```

Comment:
Looks like `Karma` does not patch `console.time`, so the results are not pushed back from the page to the Karma server.

https://github.com/karma-runner/karma/blob/0647109358362e9c10603b1e1f6894bc8b4d0bca/context/karma.js#L107-L137.

Can you please add a comment for this? at best even create an issue in the Karma repo?

Reply:
Updated the comment and created an issue.
https://github.com/karma-runner/karma/issues/3549